graphql/schemabuilder/pagination.go

```diff
@@ -434,6 +506,34 @@ func indexOfPaginationArgs(argType reflect.Type) int {
 	return -1
 }
 
+func getFuncReturnValue(fn interface{}) reflect.Type {
+	typ := reflect.TypeOf(fn)
+	return typ.Out(0)
+}
+
+func (c *connectionContext) consumeTextFilters(sb *schemaBuilder, m *method, typ reflect.Type) error {
+	c.FilterTextFields = make(map[string]*graphql.Field)
+
+	for name, fn := range m.TextFilterFuncs {
+		funcTyp := getFuncReturnValue(fn)
+		if funcTyp != reflect.TypeOf("") {
+			return fmt.Errorf("%s: Sort field must be a string", name)
```

Comment:
nit: lower lower lower 😛

Reply:
(my preference comes from https://github.com/golang/go/wiki/CodeReviewComments#error-strings)

Reply:
Done (I think)

Reply:
^ this is done btw :D